Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml

```diff
@@ -20,32 +20,51 @@
             app:popupTheme="@style/CWAToolbar.Menu"
             app:title="@string/contact_diary_overview_title" />
 
-        <TextView
-            android:id="@+id/contact_diary_overview_subtitle"
-            style="@style/subtitleMedium"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/spacing_small"
-            android:focusable="true"
-            android:text="@string/contact_diary_overview_subtitle"
-            app:layout_constraintEnd_toEndOf="@id/guideline_end"
-            app:layout_constraintStart_toStartOf="@id/guideline_start"
-            app:layout_constraintTop_toBottomOf="@id/toolbar"
-            tools:text="Tragen Sie ein, mit wem Sie sich getroffen haben und wo Sie gewesen sind." />
 
-        <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/contact_diary_overview_recyclerview"
+        <androidx.core.widget.NestedScrollView
+            android:id="@+id/scroll"
```

Comment:
maybe use a more specific id or none at all if you don't need to reference it

Reply:
The id is needed to keep the current scrolling state when navigating between the overview and day screen. Without having an id, the scrolling state wouldn't be remembered and it would jump to the top when navigation back to the overview screen. But you're right, naming is kind of meaningless, I will think about a better name :)